### ISSUE TITLE
Heisig keywords

### ISSUE THREAD

**Issue by nicolasmaia** [birtles/rikaichamp#270]
Not sure this is viable due to copyright, but it'd be nice to have the Heisig keywords listed too.

**Comment by birtles** [comment]
Yes, I'm not sure either. If you can get the data in a form I'm allowed to use, I would love to add it.

**Comment by nicolasmaia** [comment]
https://hochanh.github.io/rtk has it, I believe.

**Comment by birtles** [comment]
> https://hochanh.github.io/rtk has it, I believe.

Appears that site is in breach of copyright:

https://github.com/github/dmca/blob/master/2015/2015-11-09-Kanji.md

**Comment by nicolasmaia** [comment]
I think their breach has to do with them copying Heisig's mnemonics. I'm not sure the one-word keywords are copyrightable, really.
